File: Udemy course codes/ZTM/Data Structures and Algorithms/Searching/Linear Search.js
function search(arr, x) {

    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === x) {
            return i
        }
    }
    return -1
}


let arr = [9, 6, 4, 3, 27, 6, 533, 6657];
console.log(search(arr, 4))
console.log(search(arr, 66))

File: Udemy course codes/ZTM/Data Structures and Algorithms/Searching/ Linear Search.py
# O(n)
def search(arr, x):

    for i in range(len(arr)):

        if arr[i] == x:
            return i

    return -1


arr = [9, 6, 4, 3, 27, 6, 533, 6657]
print(search(arr, 4))
